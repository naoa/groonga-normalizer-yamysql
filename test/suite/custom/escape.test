register normalizers/yamysql

normalize NormalizerYaMySQL 'a\ab' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\bb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\fb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\tb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\vb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\\b' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\?b' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLPartofspeechHtml 'a\ab' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\bb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\fb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\tb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\vb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\\b' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLPartofspeechHtml 'a\?b' WITH_TYPES|WITH_CHECKS

