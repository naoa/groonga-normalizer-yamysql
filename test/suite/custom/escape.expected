register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQL 'a\ab' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "AAB",
    "types": [
      "alpha",
      "alpha",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQL 'a\bb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQL 'a\fb' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A\fB",
    "types": [
      "alpha",
      "others",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQL 'a\tb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQL 'a\vb' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "AVB",
    "types": [
      "alpha",
      "alpha",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQL 'a\\b' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A\\B",
    "types": [
      "alpha",
      "symbol",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQL 'a\?b' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A?B",
    "types": [
      "alpha",
      "symbol",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQLPartofspeechHtml 'a\ab' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "AAB",
    "types": [
      "alpha",
      "alpha",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQLPartofspeechHtml 'a\bb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLPartofspeechHtml 'a\fb' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A\fB",
    "types": [
      "alpha",
      "others",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQLPartofspeechHtml 'a\tb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLPartofspeechHtml 'a\vb' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "AVB",
    "types": [
      "alpha",
      "alpha",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQLPartofspeechHtml 'a\\b' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A\\B",
    "types": [
      "alpha",
      "symbol",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
normalize NormalizerYaMySQLPartofspeechHtml 'a\?b' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A?B",
    "types": [
      "alpha",
      "symbol",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
